pkg/deploy/generator/resources.go

```diff
@@ -844,6 +844,12 @@ cat >/etc/td-agent-bit/td-agent-bit.conf <<'EOF'
 	Remove_wildcard _
 	Remove TIMESTAMP
 
+[FILTER]
+	Name rewrite_tag
+	Match journald
+	Rule $LOGKIND ifxaudit ifxaudit false
```

Comment:
See Fluent Bit [doc](https://docs.fluentbit.io/manual/pipeline/filters/rewrite-tag) for more info on the `Rule` format.